samples/openthread/cli/overlay-thread_1_2_ftd.conf

```diff
@@ -0,0 +1,22 @@
+#
+# Copyright (c) 2021 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+# Configure dependencies
+CONFIG_NRF_802154_ENCRYPTION=y
+CONFIG_IEEE802154_2015=y
+CONFIG_NET_PKT_TXTIME=y
+CONFIG_NET_PKT_TIMESTAMP=y
+CONFIG_OPENTHREAD_MAC_SOFTWARE_TX_SECURITY_ENABLE=n
+CONFIG_OPENTHREAD_SLAAC=y
```

Comment:
Don't we want enable SLAAC by default?

Reply:
It's added here as a dependency for DUA. It's also added by `CONFIG_OPENTHREAD_NORDIC_LIBRARY_MASTER` for `cli` and `coprocessor` samples, since it's required for certification. The CoAP samples don't include it since they are intended to be minimal feature use cases.